tldw_Server_API/app/api/v1/endpoints/config_info.py

```
@@ -406,3 +415,333 @@ async def get_quickstart_redirect():
+            error=f"Validation timed out after {_VALIDATION_TIMEOUT_SECONDS}s",
+        )
+    except Exception as exc:
+        logger.warning(f"Provider validation failed for {provider}: {exc}")
+        return ProviderValidateResponse(
+            provider=provider,
+            valid=False,
+            error=str(exc)[:200],
+        )
+
+
+async def _validate_provider_http(
```

Ruff (0.15.7)

[warning] Too many return statements (7 > 6)

(PLR0911)